packages/kn-plugin-workflow/pkg/command/deploy.go

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/kiegroup/kie-tools/kn-plugin-workflow/pkg/common"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+)
+
+func NewDeployCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "deploy",
+		Short: "Deploy a Quarkus workflow project",
+		Long: `Deploy a Quarkus workflow project
```

Comment:
```suggestion
		Long: `
```
Not sure if we need to repeat ourselves here. I can't remember what cobra prints, but if this is the pattern feel free to ignore this and the other comment.